python/ray/air/_internal/usage.py

```diff
@@ -115,3 +117,29 @@ def tag_scheduler(scheduler: "TrialScheduler"):
     assert isinstance(scheduler, TrialScheduler)
     scheduler_name = _find_class_name(scheduler, "ray.tune.schedulers", TUNE_SCHEDULERS)
     record_extra_usage_tag(TagKey.TUNE_SCHEDULER, scheduler_name)
+
+
+def tag_env_vars() -> bool:
```

Comment:
Can we name this after `ray_env_var`s or similar so its clearer that this is capturing only Ray's env vars?

Reply:
Ah good point. I think `tag_ray_air_env_vars` is even better.